bundles/org.openhab.binding.pushbullet/README.md

```diff
@@ -0,0 +1,162 @@
+|------------|------------------------------------------------------------------|----------|----------|
+| token      | Pushbullet [API token](#obtaining-an-api-key) to send to devices | Yes      | False    |
+| name       | Explicit Name, for later use when the bot can receive messages   | No       | True     |
+| apiUrlBase | Address of own Pushbullet server, for testing purposes           | No       | True     |
+
+```java
+Thing pushbullet:bot:r2d2 "R2D2" @ "Somewhere" [ token = "verysecretwonttellyou" ]
+
+```
+
+## Channels
+
+| Channel ID | Channel Description                             | Supported item type  | Advanced |
+|------------|-------------------------------------------------|----------------------|----------|
+| recipient  | for later use when the bot can receive messages | String               | False    |
+| title      | for later use when the bot can receive messages | String               | False    |
+| message    | for later use when the bot can receive messages | String               | False    |
+
+## Rule Action
+
+This binding includes rule actions for sending notes.
+Two different actions available:
+
+* `sendPushbulletNote(String recipient, String messsage)`
+* `sendPushbulletNote(String recipient, String title, String messsage)`
+
+Since there is a separate rule action instance for each `bot` thing, this needs to be retrieved through `getActions(scope, thingUID)`.
+The first parameter always has to be `pushbullet` and the second is the full Thing UID of the bot that should be used. 
+Once this action instance is retrieved, you can invoke the action method on it.
+
+Examples:
+
+```
+val actions = getActions("pushbullet", "pushbullet:bot:r2d2")
+val result = actions.sendPushbulletNote("someone@example.com", "R2D2 talks here...", "This is the pushed note.")
+```
+
+## Full Example
+
+_Provide a full usage example based on textual configuration files (*.things, *.items, *.sitemap)._
+
+pushbullet.things:
+
+```java
+Thing pushbullet:bot:r2d2 "R2D2" @ "Somewhere" [ token = "verysecretwonttellyou" ]
+
+```
+
+pushbullet.items
+
+```java
+Switch Pushbullet_R2D2_Button  "Pushbullet Action bot R2D2"
+```
+
+pushbullet.sitemap
+
+```java
+sitemap pushbullet label="Pushbullet"
+{
+    Switch item=Pushbullet_R2D2_Button
+}
+```
+
+pushbullet.rules
+
+```java
+rule "Pushbullet R2D2 changed"
+when
+    Item Pushbullet_R2D2_Button changed
+then
+    logInfo(filename, "Button R2D2 changed - OH2...")
+
+    if (Pushbullet_R2D2_Button.state == ON)
+    {
+        sendCommand(Pushbullet_R2D2_Button, OFF)
+
+        val actions = getActions("pushbullet", "pushbullet:bot:r2d2")
+        logInfo(filename, "Actions for 'R2D2' are: " + actions)
+
+        if (actions != null)
+        {
+            val result = actions.sendPushbulletNote("someone@example.com", "Title R2D2 OH2", "This has been sent by the new R2D2 bot")
+            logInfo(filename, "Result of send action is: " + result)
+        }
+    }
+end
+```
+
+
```

Comment:
Could you maybe remove some of the newlines here

Reply:
Done.